include/qdmi/constants.h

```diff
@@ -512,6 +512,15 @@ enum QDMI_OPERATION_PROPERTY_T {
   QDMI_OPERATION_PROPERTY_DURATION = 3,
   /// `double` The fidelity of an operation.
   QDMI_OPERATION_PROPERTY_FIDELITY = 4,
+  /**
+   * @brief @ref QDMI_Operation_Scope The scope of the operation.
+   * @details The scope of an operation indicates whether the operation
+   * executes on individual sites or on all sites this operation can be applied
+   * to.
+   * @note This property is mainly required for neutral atom devices that offer
+   * apart from local operations also global operations.
```

Comment:
```suggestion
   * @note This property is mainly required for neutral atom devices that offer
   * global operations in addition to local operations.
```
The original wording feels awkward.